### ISSUE TITLE
Maven dependency

### ISSUE THREAD

**Issue by CesarChodun** [stephengold/Minie#8]
Importing the project via: 
` <dependency>
            <groupId>com.github.stephengold</groupId>
            <artifactId>Minie</artifactId>
            <version>1.7.0</version>
  </dependency>`
doesn't work (in Maven).  It would be great if the tutorial could say how to import this project to maven.

**Comment by stephengold** [comment]
In what way doesn't it work? Does importing the project as you described do anything at all? Is there an error message?

**Comment by CesarChodun** [comment]
Yes, there is an error message: "Cannot resolve com.github.stephengold:Minie:1.7.0"

**Comment by stephengold** [comment]
I rarely use Maven. I suspect you need to add the "JCenter" repository to your POM or user profile.

If added to the POM, it should look something like this:
```xml
<repositories>
    <repository>
      <id>jcenter</id>
      <url>https://jcenter.bintray.com/</url>
    </repository>
</repositories>
```

Let me know if that works for you. If it does, I'll add it to the documentation.

**Comment by CesarChodun** [comment]
It works now. Thank you for helping me :)

**Comment by stephengold** [comment]
Resolved in master branch at 3fe8f870.
